packages/editor/src/components/blocks/datatable/DataTable.css

```diff
@@ -70,3 +70,16 @@
 .canvas[data-help-paddings='false'] .block-table .block-table__columns .draggable {
   border-radius: 0;
 }
+
+.block-table__columns:has(.block-search__input) .block-search__placeholder {
```

Comment:
I deliberately kept the placeholder span similar to the search-span and also added some text inside the placeholder span to ensure that the height matches exactly with the actual column containing the search field. If I had simply set a height, it wouldn’t have been perfectly the same.